content/quickstart/_index.md

```diff
@@ -5,36 +5,24 @@ menu:
     weight: 1
 ---
 
-Pulumi is a platform for building and deploying cloud infrastructure and applications in your favorite language on any
-cloud.
-
-Describe your resources in code -- VMs, networks, databases, containers, Kubernetes clusters & workloads, serverless functions --
-and the CLI safely and reliably manages your cloud resources using an infrastructure-as-code approach.
-
-Create, share, and reuse powerful cloud packages.
-
-Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and is free to use.
-
-## Download and Install
-
-To get started, first download and install the CLI:
-
-<a href="{{< relref "install.md" >}}"><button class="button primary small">INSTALL PULUMI</button></a>
+Get started provisioning infrastructure with Pulumi.
 
 ## Choose Your Cloud
```

Comment:
It's the right call not showing Openstack, etc here anymore. But should we have a link beneath these clouds, something like

> Pulumi supports other resource providers not listed here. If you're looking for a cloud not listed here,
> please see xxxxxxxxx for information on how to use it.

I'm not sure what xxxxxxxxx is, some index to our providers? (Does such a thing exist?)

I definitely *do not* want to clutter the getting started flow, so I'd put it at the very end beneath the icons, and try not to draw attention to it.

Reply:
Agreed. Chris gave me the same feedback.

Reply:
I ended up just linking to the Supported Clouds page for now, which includes OpenStack, and mentions other resource providers on GitHub.

<img width="545" alt="Screen Shot 2019-06-14 at 12 35 52 PM" src="https://user-images.githubusercontent.com/710598/59533628-f4885e00-8ea0-11e9-8e1f-a7435c9ac505.png">